src/store/state.ts

```diff
@@ -20,7 +20,7 @@ export const state: StateIF = {
   entityState: null,
   entityStatus: null,
   entityFoundingDate: null,
-  goodStanding: null,
+  goodStanding: true,
```

Comment:
the default state would be loaded for IAs as there was no business and null evaluated to false in the check for inGoodStanding

Reply:
I think the store should be "empty" initially (or this can have future side-effects when something is assumed incorrectly).

How about setting Good Standing here (with a suitable comment) instead?
https://github.com/bcgov/business-filings-ui/blob/d68faa5c447162731dd9f8abfb8b763d2dd4d8fa/src/App.vue#L600

Reply:
Done, good catch @ChrisGabel23

Reply:
LGTM now.